portfolio/src/main/webapp/style.css

```diff
@@ -52,8 +64,63 @@ footer {
   background: black;
 }
 
+section {
+  animation: fadeIn 1s ease-in-out;
+}
+
+.section-content {
+  padding: 6rem;
+}
+
+.education-post {
+  align-items: center;
+  display: flex;
+  flex-wrap: wrap;
+  margin: 3rem 3rem;
+}
+
+.education-detail {
+  margin-left: 2rem;
+}
+
+#education,
```

Comment:
This is a lot of different ids - maybe you could give each of these a class so that it makes sense why they're grouped together?

Reply:
Oh, those are the sections of the portfolio, I used ids to be able to select them more easily with javascript, then I can switch between sections with the navbar/terminal. Is there another approach that you'd recommend me instead of doing that?

Reply:
Resolved via chat.